ApollosProject/apollos-prototype
# Use apollos plugin to fix max node issues

## DESCRIPTION

### What does this PR do, or why is it needed?

![Uploading 2019-09-04 21.23.41.gif…]()
Gets around a common OData node limit issue by using a custom (filterable) endpoint when fetching more than one item. This should have no change in API functionality, besides queries that currently error out, _not_ erroring out. 

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

1. Open up a series with more than 10 (or so) children.
2. Visit one of the children
3. The child should have their siblings listed below the content fold.

### What automated tests did you write?

Tests over new functionality.

## TODO:

- [x] Closes #729

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
